index.rst

```diff
@@ -6,20 +6,423 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
+Overview
+========
+
+This document describes the procedure to pump and cool down LATISS from ambient pressure to its operational temperatures. 
+It also outlines how to perform a controlled warm-up in the case of servicing or programmed power cuts. 
+
+Pumping LATISS to vacuum
+========================
+
+Under atypical circumstances, LATISS needs to be warmed up to ambient temperature. 
+Large periods of power outage, severe earthquakes (`and different levels of recovery based on severity <https://tstn-027.lsst.io>`__), or technical intervention may have necessitated warm-up.
+Pumping down, cooling, and powering up LATISS takes at least two days, and must be organized by ticket on the summit calendar and supported by daytime engineering support. 
+Mario Rivera, Craig Lage, and Alysha Shugart are points of contact for understanding and initiating the procedure. 
+
+Getting started
+---------------
+
+AuxTel must be under LOTO or the `E-STOP <https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.html>`__ must be engaged to perform this operation. 
```

Comment:
Are we still using the E-stop?

Reply:
Both, until I can publish the LOTO procedure for everyone to use. 👍